docs/developer-docs/latest/setup-deployment-guides/configurations/optional/hooks.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Hooks - Strapi Developer Documentation
+description:
+---
+
+<!-- TODO: update SEO -->
+
+# Hooks
```

Comment:
I'm not sure if this should still exist since we basically depreciated the v3 hooks system, everything should just be a plugin now.

Reply:
Indeed, thanks Derrick! 
I planned to have a specific PR to remove hooks.

FYI this PR is only about splitting files and reformating things, not updating the content (will come later with other PRs 🙂 (added a note in the PR description — see screenshot👇 :D )

![Screenshot 2021-10-04 at 21 49 25](https://user-images.githubusercontent.com/4233866/135915086-e982213e-485d-48be-819e-801672c08593.png)
)

Reply:
Ah sorry, I got too far ahead of myself and I should have read that.